rj_gameplay/rj_gameplay/role/marker.py

```diff
@@ -0,0 +1,29 @@
+import stp
+from rj_gameplay.skill import mark
+
+from rj_msgs.msg import RobotIntent
+
+
+class MarkerRole(stp.role.Role):
```

Comment:
can you add a docstring here explaining what the role does? (I know in this case it's very simple)